flavors/python-3.8-conda-minimal-EXASOL-6.2.0/flavor_base/udfclient_deps/packages/conda_packages

```diff
@@ -0,0 +1,6 @@
+libprotobuf|3.20.1
+zeromq|4.3.4
+openssl|1.1.1o
+nss|3.77
+cppzmq|4.8.1
+_openmp_mutex|4.5=2_kmp_llvm # We need this, such that we can later install libblas with mkl, which is for example needed by pytorch
```

Comment:
Hm, I had the understanding that the step `udfclient_deps` installs only the runtime dependencies for the udf-client. Then shouldn't be _openmp_mutex rather a dependency of `language_deps`?